<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportAnnotationsWizardPraatPage</class>
 <widget class="QWizardPage" name="ExportAnnotationsWizardPraatPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Export Annotations to Praat Textgrids</string>
  </property>
  <property name="subTitle">
   <string>In this step you can define the structure of the exported TextGrids.</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupboxSpeakerStrategy">
     <property name="title">
      <string>Speaker strategy</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QRadioButton" name="optionOneSpeakerPerFile">
        <property name="text">
         <string>Create a separate TextGrid file for each speaker</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="optionAllSpeakers">
        <property name="text">
         <string>Include all speakers in the same TextGrid</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupboxLevelsAttributes">
     <property name="title">
      <string>Annotation Levels and Attributes to export</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QTreeView" name="treeviewLevelsAttributes"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QGroupBox" name="groupboxTextgridStructure">
     <property name="title">
      <string>Structure of exported TextGrids</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="1">
       <widget class="QToolButton" name="buttonMoveDown">
        <property name="text">
         <string>Move Down</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QToolButton" name="buttonMoveUp">
        <property name="text">
         <string>Move Up</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QTableView" name="tableviewTextgridStructure"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="groupBoxFilenameTemplate">
     <property name="title">
      <string>Filenames template</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLineEdit" name="editFilenameTemplate"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelFilenameTemplate">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>You may use the following fields: $CorpusID, $CommunicationID, $SpeakerID, $AnnotationID and $RecordingID.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>editFilenameTemplate</tabstop>
  <tabstop>treeviewLevelsAttributes</tabstop>
  <tabstop>tableviewTextgridStructure</tabstop>
  <tabstop>buttonMoveUp</tabstop>
  <tabstop>buttonMoveDown</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
